internal/lsp/server.go

```diff
@@ -499,6 +499,52 @@ func (l *LanguageServer) StartCommandWorker(ctx context.Context) { // nolint:mai
 
 				// handle this ourselves as it's a rename and not a content edit
 				fixed = false
+			case "regal.debug":
+				if l.clientIdentifier != clients.IdentifierVSCode {
+					l.logError(errors.New("regal.debug is only supported in VSCode"))
+
+					break
+				}
+
+				if len(params.Arguments) != 3 {
```

Comment:
I don't see us reading arg `2` anywhere. Looks like it's the line number for `regal.eval`, which I suppose we don't need for debug. Do we still need to assert its presence?

Reply:
No, you're right, we don't need to assert this here, but we're using the same outputs for both code lenses... and we'll want to check this to avoid getting out of bounds errors in case there aren't at least 2 args.